### ISSUE TITLE
YouTubeCenter in combination with Adblock Plus cause delay on youtube

### ISSUE THREAD

**Issue by JonasNo** [YePpHa/YouTubeCenter#1848]
The cpu use is high. Can't do anything with for a few seconds.

**Comment by SuperSajuuk** [comment]
Yeah, Adblock Plus + YT Center freeze the page completely for a bit. Disabling the Adblock Plus extension fixes the freezes, so I guess it's because YT Center also has a "remove ads" option.

**Comment by Piter432** [comment]
@JonasNo Use uBlock Origin instead of Adblock Plus. uBlock Origin is better
than Adblock Plus.

2015-05-28 14:10 GMT+02:00 SuperSajuuk <notifications@github.com>:

**Comment by muyte** [comment]
I don't use adblock plus and I get the 100% cpu freezes many times per day.

**Comment by DAOWAce** [comment]
@Piter432 I thought you were another preference troll, but then I checked it out.

https://github.com/gorhill/uBlock/wiki/uBlock-vs.-ABP:-efficiency-compared

You're right.

I've switched over to it; thanks for mentioning it.

Feels like the page loads slightly faster, but there's still a big delay on page load compared to with YTC disabled.
